src/AUDIT_TRAIL_2.md

```diff
@@ -0,0 +1,49 @@
+# Audit trail for Mangrove v2
+
+The goal of Mangrove v2 is constant-time offer insertion.
```

Comment:
and offer update. Should also be updated in the following text.

Reply:
But it's specifically insertion (whether as part of an update) that used to not take constant time, so I think it's ok to just mention that?

Reply:
To the casual reader I don't think update/retract~=insertion

Reply:
Ok. I think it would be clearer mentioning this, but I don't feel strongly about this and we need to move quickly, so 🤷🏻‍♂️

Reply:
added ref to update f9af61f1c93c45975590d4cfe47985a8a513dc38